.circleci/config.yml

```
@@ -109,6 +109,20 @@ jobs:
                 shell: /bin/bash -eo pipefail
                 command: |
                   bash <(curl -s https://codecov.io/bash)
+  build:
+    executor: golang
+    steps:
+      - install-deps
+      - prepare
+      - go/mod-download
+      - run:
+          command: make -j2 build
+      - store_artifacts:
+          path: lotus
+          destination: lotus
```

Having some way to tell what the platform is would be nice, something like:
```suggestion
          destination: lotus.amd64.linux
```